node/task.ts

```diff
@@ -420,7 +420,7 @@ export function getEndpointAuthorization(id: string, optional: boolean): Endpoin
         setResult(TaskResult.Failed, loc('LIB_EndpointAuthNotExist', id));
     }
 
-    debug(id + ' exists ' + (aval !== null));
+    debug(id + ' exists ' + (!!aval));
```

Comment:
I suggest `aval !== undefined` instead of `!!aval`, because I think that's closer to the intended meaning of the code and has less surprising semantics: for example, `"" !== undefined` is `true`, but `!!""` is `false`.

Also, it might be worth mentioning in the PR description/commit message body _why_ this change is made (namely, because `aval` has type `string | undefined` and thus can't be `null`).

Reply:
Thanks for the detailed comment. I think, in our case, it will be better to use !!aval for more strong verification, it will allow us to check the empty string also.